crates/rust-analyzer/src/diagnostics/to_proto.rs

```diff
@@ -118,6 +118,7 @@ fn map_rust_child_diagnostic(
                 changes: Some(edit_map),
                 document_changes: None,
             }),
+            is_preferred: Some(true),
```

Comment:
This is correct though right?

Reply:
I would think so, but i haven't re-checked underlying semantics of our conversion and original JSON